cmd/admin_user_create.go

```diff
@@ -67,6 +67,8 @@ var microcmdUserCreate = &cli.Command{
 }
 
 func runCreateUser(c *cli.Context) error {
+	setting.LoadSettings()
```

Comment:
It's better to move this line to line 86 and have some tests in `admin_user_create_test.go`.

Reply:
This is not right, because then some settings will be loaded twice.